radar-auth/build.gradle

```diff
@@ -1,19 +1,6 @@
-plugins {
-    // Apply the java plugin to add support for Java
-    id 'java-library'
-    id 'pmd'
-    id 'checkstyle'
-    id 'maven-publish'
-}
-
-sourceCompatibility = 1.8
-
-ext {
-    jacksonVersion = '2.9.8'
-    okhttpVersion = '3.13.1'
-
-    moduleDescription = 'Library for authentication and authorization of JWT tokens issued by the RADAR platform'
-}
+ext.jacksonVersion = '2.9.8'
+ext.okhttpVersion = '3.13.1'
+ext.moduleDescription = 'Library for authentication and authorization of JWT tokens issued by the RADAR platform'
```

Comment:
In other projects, I've been moving away from `ext.moduleDescription`, and just using the builtin `description` variable. That should then also be used in the `bintray` and `publications` definitions.